roles/vault/tasks/vault.yml

```diff
@@ -30,10 +30,10 @@
   run_once: True
   no_log: True
   when: not vault_init_status.json.initialized
-  register: vault_init_output
+  register: vault_keys
 
 - name: Print vault keys
   debug:
-    var: vault_init_output
+    var: vault_keys
   when: not vault_init_status.json.initialized
```

Comment:
Maybe we should also not do this :)

Reply:
From one perspective we shouldn't, but then I see two options:
1) Write in docs that you need to reuse that variable to know what are the unseal keys
2) Do a when: in the print task, default to no and write in docs that by default you need to reuse the variable or use -e init_print_keys=True